_posts/2017-07-09-P3-Quality.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+asset-type: post
+name: P3 Quality
+title: 'P3 Quality'
+date: 2017-07-09 11:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-07-09-P3-Quality.jpg
+tags:
+- craftsmanship
+- quality
+
+---
+
+Do we know what is software quality? Is it measurable? If we want to improve the quality, do we know what to focus on?
```

Comment:
instead of `Do we know what is software quality?`
I would write `What is software quality?`
`If we want to improve the quality, do we know what to focus on?` -> `What should we focus on when we want to improve quality?` or `What does it mean to improve software quality?`